src/app/themes/theme.js

```diff
@@ -201,6 +201,47 @@ const theme = createMuiTheme({
         width: '100%',
       },
     },
+
+    MuiTablePagination:{
+      root:{
+        display: 'flex',
+        alignItems: 'center',
+        alignContent: 'center',
+        justifyContent: 'center',
+        flexWrap: 'wrap',
+        fontSize:'1em',
```

Comment:
Linting rules says there should be a space after the `:` in a class object. There are a few others like that in this file that will eventually need to be fixed, so am just pointing it out in this one

```suggestion
        fontSize: '1em',
```